src/setupTests.ts

```diff
@@ -10,6 +10,8 @@ if (typeof global.TextEncoder === 'undefined') {
   global.TextDecoder = TextDecoder as typeof global.TextDecoder
 }
 
+global.ResizeObserver = require('resize-observer-polyfill')
```

Comment:
This should be explicitly added to devDependencies. Right now it's just included as a sub-dep of widgets.

Should we be including this in the build? Or can we always depend on resizeObserver being defined?

Reply:
I added it to dev dependencies but we still need this line here for the tests to pass.

Reply:
that's expected - it's because jest is running its own DOM environment so it's not actually an up-to-date browser

Reply:
You shouldn't require unless you need to do so conditionally. Split this into two lines:

```
import ResizeObserver from 'resize-observer-polyfill'
...
global.ResizeObserver = ResizeObserver
```